### PR TITLE
Add fees

### PR DESCRIPTION
**Pull request by fna**
@amymok , as far as I know Eric hasn't implemented changes on his end, so don't merge this just yet

### CODE REVIEW

File: ratechecker/models.py [hunk 1]
```diff
@@ -137,3 +137,21 @@ class Rate(models.Model):
...
+class Fee(models.Model):
+    fee_id = models.AutoField(primary_key=True)
+    plan = models.ForeignKey(Product)
+    product_id = models.IntegerField()
+    state_id = USStateField()
+    lender = models.CharField(max_length=16)
+    single_family = models.BooleanField(default=True)
+    condo = models.BooleanField(default=False)
+    coop = models.BooleanField(default=False)
+    origination_dollar = models.DecimalField(max_digits=8, decimal_places=4)
```
Comment (amymok) [inline ratechecker/models.py]: If this is currency, decimal_places should be 2

**Comment by amymok** [comment]
Can you add to the documentation also?  Thanks.

**Comment by amymok** [comment]
Can you also take a look at the test coverage as well?  Thanks!

**Comment by fna** [comment]
@amymok I've added a couple of tests related to fees here, but I'll work on test coverage in another branch; this branch won't be updated and is ready to be merged

**Pull request merged**